devops/.env.sample

```diff
@@ -1,12 +1,17 @@
+# Management infrastructure
 ARM_TENANT_ID=__CHANGE_ME__
 ARM_SUBSCRIPTION_ID=__CHANGE_ME__
-ARM_CLIENT_ID=__CHANGE_ME__
-ARM_CLIENT_SECRET=__CHANGE_ME__
```

Comment:
@christoferlof these are needed for deploying bundles, such as:

https://github.com/microsoft/AzureTRE/blob/develop/workspaces/azureml_devtestlabs/azure.json

Reply:
They are still in the .sample but further down, in an attempt to group the variables based on their usage. If we're requiring two SPs for the quickstart scenario, we need to add that obviously.